apps/ws/src/readiness/readiness.module.ts

```diff
@@ -0,0 +1,47 @@
+import { Module, Provider } from '@nestjs/common';
+import {
+  DalServiceHealthIndicator,
+  IHealthIndicator,
+  ReadinessService,
+  WebSocketsQueueService,
+  WebSocketsQueueServiceHealthIndicator,
+} from '@novu/application-generic';
+import { WSServerHealthIndicator } from '../socket/services';
+import { SharedModule } from '../shared/shared.module';
+import { SocketModule } from '../socket/socket.module';
+
+export const indicatorList = {
+  provide: 'INDICATOR_LIST',
+  useFactory: (
+    wsServerHealthIndicator: WSServerHealthIndicator,
+    webSocketsQueueServiceHealthIndicator: WebSocketsQueueServiceHealthIndicator,
+    dalServiceHealthIndicator: DalServiceHealthIndicator
+  ) => {
+    const indicators: IHealthIndicator[] = [
+      wsServerHealthIndicator,
+      webSocketsQueueServiceHealthIndicator,
+      dalServiceHealthIndicator,
+    ];
+
+    return indicators;
+  },
+  inject: [WSServerHealthIndicator, WebSocketsQueueServiceHealthIndicator, DalServiceHealthIndicator],
+};
+
+const PROVIDERS: Provider[] = [
+  WSServerHealthIndicator,
+  WebSocketsQueueServiceHealthIndicator,
+  DalServiceHealthIndicator,
+
+  WebSocketsQueueService,
+
+  indicatorList,
+  ReadinessService,
+];
+
+@Module({
+  imports: [SharedModule, SocketModule],
```

Comment:
If we import the needed indicator services here separately I noticed that Nest will create separate instances, which is not the desired behavior because we want to validate the health of the connection that the APP is using.

In order to get the same instance we need to use the module that initializes the services.
* for DalServiceHealthIndicator we import SharedModule.
* for the wsServerHealthIndicator we import SocketModule.